packages/common/src/events/_internal/EventUiSchemaEdit.ts

```diff
@@ -212,6 +212,14 @@ export const buildUiSchema = async (
                 context.portal.name,
                 context.hubRequestOptions
               ),
+              locationNameRequired: true,
+              messages: [
```

Comment:
I went ahead and hid the error message that's shown as a result of the "then" condition not being met...but you can unhide it/overwrite the default text if desired